src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hey there! I'm Duke\n" + "What would you like to do?" + "\n");
+
+        List<Task> taskList = new ArrayList<>();
+
+        //Takes user input
+        Scanner input = new Scanner(System.in);
+
+        //this do-while loop asks users for input, can add items to lists and print the list
+        while (!input.hasNext("bye")) {
+            String userInput = input.nextLine();
+            String userCommand = userInput.split("\\s", 2)[0];
+            int stringLen = userInput.length();
+
+            //adds a todo item to the list
+            if (userCommand.equals("todo")) {
+                String noCommand = userInput.split("\\s", 2)[1];
```

Comment:
use variable names that is less confusing. noCommand can mean 2 things, 1) no command and 2) number of command

Reply:
Got it, will change it to make it clearer!